contracts/tokens/SecurityToken.sol

```diff
@@ -355,6 +426,7 @@ contract SecurityToken is ISecurityToken {
     function mint(address _investor, uint256 _amount) public onlyModule(STO_KEY, true) checkGranularity(_amount) returns (bool success) {
         adjustInvestorCount(address(0), _investor, _amount);
         require(verifyTransfer(address(0), _investor, _amount), "Transfer is not valid");
+        adjustTotalSupplyCheckpoints();
         totalSupply_ = totalSupply_.add(_amount);
         balances[_investor] = balances[_investor].add(_amount);
         emit Minted(_investor, _amount);
```

Comment:
Shouldn't `adjustBalanceCheckpoints` be called here as well? If not an account would have a different amount of tokens than what balanceOfAt will return.

Reply:
Agreed - added.